### PR TITLE
fixing typos while searching for create fix

### PR DESCRIPTION
**Pull request by jasimper**
A mix of me fixing typos and Stan taking over my computer to fix database stuff.

### CODE REVIEW

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -1,3 +0,0 @@
-<div data-js="expenses-index">
```
Comment (mcmillanator) [inline app/views/home/index.html.erb]: I think Ryan was using this div to print react elements to the screen.

Comment (jasimper) [inline app/views/home/index.html.erb]: Sorry, I didn't see any references to it, only the index file in the expenses folder.

Comment (mcmillanator) [inline app/views/home/index.html.erb]: I think it's something we're planning on using, and isn't yet in use.  Let's check with Ryan before closing this PR.  @ryanmcgov Let us know.  :)

Comment (ryanmcgov) [inline app/views/home/index.html.erb]: Yea, I was just using it to have a place to render the form for the first time.

**Pull request merged**